dronecan_gui_tool/setup_window.py

```diff
@@ -105,6 +105,11 @@ def list_ifaces():
         for x in mifaces:
             out[x] = x
 
+        from can import detect_available_configs
```

Comment:
I think we need a try except around this, to cope with systems without the can python module

Reply:
i've added the exception check, can you test?

Reply:
Tested with try except without python-can installed on windows seems to be working fine.